controllers/src/test/scala/cromwell/pipeline/controller/ProjectConfigurationControllerTest.scala

```diff
@@ -81,18 +81,19 @@ class ProjectConfigurationControllerTest extends AsyncWordSpec with Matchers wit
     "get configuration by project id" should {
       "return configuration by existing project id" in {
         when(configurationService.getLastByProjectId(projectId, accessToken.userId))
-          .thenReturn(Future.successful(Some(configuration)))
+          .thenReturn(Future.successful(configuration))
         Get(s"/projects/${projectId.value}/configurations") ~> configurationController.route(accessToken) ~> check {
           status shouldBe StatusCodes.OK
           entityAs[ProjectConfiguration] shouldBe configuration
         }
       }
 
-      "return message about no project with this project id" in {
-        when(configurationService.getLastByProjectId(projectId, accessToken.userId)).thenReturn(Future.successful(None))
+      "return Configuration not found message" in {
+        when(configurationService.getLastByProjectId(projectId, accessToken.userId))
+          .thenReturn(Future.failed(NotFound()))
         Get(s"/projects/${projectId.value}/configurations") ~> configurationController.route(accessToken) ~> check {
           status shouldBe StatusCodes.NotFound
-          entityAs[String] shouldBe s"There is no configuration with project_id: ${projectId.value}"
+          entityAs[String] shouldBe s"Configuration not found"
```

Comment:
Can't we preserve this message? I mean both options are fine, but I don't see any reason why we need to change it

Reply:
done.